docs/NEEM-handbook/content/acquisition/acquisition.tex

```diff
@@ -53,6 +53,21 @@ \section{Data Structure}
 	\item[\textbf{triples}] The triples collection contains all triples which were asserted into the knowledge base during run time.
 \end{description}
 
+\subsection{Triple data as JSON object}
+	One can provide triple data into JSON format logged in their experiment, where triples are defined in standard subject, predicate and object format. Subject and objects can be defined with valid IRI, which is pointing to concepts or individuals defined by ontology. An individual name is the combination of concept name and random string connected with an underscore. An object can be also other data types, such as a number, string, boolean, etc. Whereas, the predicate is defined by IRI pointing to property concepts mentioned in  ontology. One can also provide additional time scope fields `since` and `until` to indicate that the given triple is valid for the given time scope. An example of such JSON object is given below:
```

Comment:
"Subject and objects" singular/plural mix-up
either both singular or both plural